com.diffingo.fwbackups.json

```diff
@@ -0,0 +1,67 @@
+{
+  "app-id": "com.diffingo.fwbackups",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "43",
+  "sdk": "org.gnome.Sdk",
+  "command": "fwbackups",
+  "build-options": {
+    "build-args": [
+      "--share=network"
```

Comment:
build must run off-line. This is not allowed.

Reply:
Thanks for the fast review -- reworked the installation to source wheels from pypi with reqs2flatpak. Let me know how it looks now.